examples/network/edgeStyles/selfReference.html

```diff
@@ -22,34 +22,73 @@
     <script type="text/javascript">
 
         // create an array with nodes
+        const multilineString1 = [
+            "Node has selfReference edge,",
+            "which angle (position) is provided.",
+            "Property renderBehindTheNode allows to draw parts,",
+            "which are not overlaping with the node."
+        ].join("\n");
+
+        const multilineString2 = [
+            "I'm an alligator,",
+            "I'm a mama-papa coming for you.",
+            "I'm the space invader,",
+            "I'll be a rock 'n' rollin' bitch for you."
+        ].join("\n");
+
         var nodes = new vis.DataSet([
             {
                 id: 1,
                 label:
-                    "Node has selfReference edge,\nwhich angle (position) is provided.\nProperty renderBehindTheNode allows to draw parts,\nwhich are not overlaping with the node",
+                    multilineString1,
```

Comment:
I think this should've been multilineString2. Also the variable name is just lazy and since it's used just once inline it here, there's no need for a variable in this case.